Configurations/config_default.json

```diff
@@ -9,12 +9,6 @@
   "encoders": [
```

Comment:
I see that there are quite a few changes in the default config file. Can you make a small list of what's changed, then when this get's merged and released I can include it in the release notes.